efel/units/units.json

```diff
@@ -0,0 +1,129 @@
+{
+    "ADP_peak_amplitude": "mV",
+    "ADP_peak_values": "mV",
+    "AHP_depth": "mV",
+    "AHP_depth_abs": "mV",
+    "AHP_depth_abs_slow": "mV",
+    "AHP_depth_diff": "mV",
+    "AHP_depth_from_peak": "mV",
+    "AHP_depth_last": "mV",
+    "AHP_depth_slow": "mV",
+    "AHP_slow_time": "constant",
+    "AHP_time_from_peak": "ms",
+    "AHP_time_from_peak_last": "mV",
+    "AP1_peak": "mV",
+    "AP2_AP1_begin_width_diff": "ms",
+    "AP2_AP1_diff": "mV",
+    "AP2_AP1_peak_diff": "mV",
+    "AP_Amplitude": "mV",
+    "AP_Amplitude_change": "constant",
+    "AP_amplitude_diff": "mV",
+    "AP_amplitude_from_voltagebase": "mV",
+    "AP_begin_time": "ms",
+    "AP_begin_voltage": "mV",
+    "AP_begin_width": "ms",
+    "AP_duration": "ms",
+    "AP_duration_change": "constant",
+    "AP_duration_half_width": "ms",
+    "AP_duration_half_width_change": "constant",
+    "AP_fall_rate": "V",
+    "AP_fall_rate_change": "constant",
+    "AP_fall_time": "ms",
+    "AP_peak_downstroke": "V",
+    "AP_peak_upstroke": "V",
+    "AP_phaseslope": "1",
+    "AP_phaseslope_AIS": "1",
+    "AP_rise_rate": "V",
```

Comment:
Same ?

Reply:
Good catch, fixing it by updating the parser.

Reply:
Done